RELEASE.rst

```diff
@@ -0,0 +1,28 @@
+Release
+=======
+
+This project is released on PyPI as `coveralls`_.
+
+To cut a new release, ensure the latest master passes all tests. Then, create a release commit:
+
+1. Update the :code:`CHANGELOG.rst` with the new version.
+2. Bump the version number in :code:`setup.py`.
+3. Tag that commit with the version number (:code:`git tag x.y.z`).
```

Comment:
Auto-changleogs is a good idea. Personally, I've been using the [conventional-changelog format](https://github.com/conventional-changelog/conventional-changelog/blob/4bf7d8df29f7be29552b82ec2b7112fad0c39a3e/conventions/angular.md) since it seems to be gaining popularity as a standard and tools like [clog](https://github.com/clog-tool/clog-cli) for the generation, but any option works well here.

I'd also appreciate if we switched to strict [semver](http://semver.org/) rather than the varying tags that have been used so far.